lib/chef/knife/customization_helper.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Author:: Malte Heidenreich (https://github.com/mheidenr)
+# License:: Apache License, Version 2.0
+#
+
+module CustomizationHelper
+  def CustomizationHelper.wait_for_sysprep(vm, vim_connection, timeout, sleep_time)
```

Comment:
I use the Namespace idea of module functions, there you directly see where the methods come from. (like the Math module)